src/shiftedNormL0Box.jl

```diff
@@ -59,20 +62,18 @@ shifted(
   selected::AbstractArray{T} = 1:length(xk),
 ) where {R <: Real, T <: Integer} = ShiftedNormL0Box(h, xk, zero(xk), -Δ, Δ, false, selected)
 shifted(
-  ψ::ShiftedNormL0Box{R, T, V0, V1, V2, V3, V4},
+  ψ::ShiftedNormL0Box{R, V0, V1, V2},
```

Comment:
Not sure I understand this line. What about parameters V3, V4 and VI?

Reply:
They are not needed, and I don't think that removing them slows the function. I left V0, V1, V2 so that we can ensure that they are `<:AbstractVector{R}`.